src/Integration.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -337,7 +338,7 @@ public CSharpVBBindingConfigProvider CreateTestSubject()
                     .ReturnsAsync(InactiveRulesResponse);
 
                 sonarQubeServiceMock
-                    .Setup(x => x.GetAllPropertiesAsync(It.IsAny<string>(), It.IsAny<CancellationToken>()))
```

Comment:
Updated the mock to only return the properties if the project key is correctly passed through.